buildenv/jenkins/jobs/infrastructure/branchSplit

```diff
@@ -0,0 +1,121 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+// Assumes OPENJ9_SHA, OMR_SHA, and NEW_BRANCH_NAME are passed in Jenkins parameters
+
+HTTP = 'https://'
+OMR_REPO = 'github.com/AdamBrousseau/openj9-omr.git'
+OPENJ9_REPO = 'github.com/AdamBrousseau/openj9.git'
+
+DEFAULT_SPLIT_OPENJ9 = 'master'
+DEFAULT_SPLIT_OMR = 'openj9'
```

Comment:
Can you clarify the meaning of `DEFAULT_SPLIT_`?  It might just be a naming issue but I'm not 100% sure what this means.

Reply:
If a sha is not provided, we default to splitting from tip of master/openj9